docs/sdk/authentication/session-sigs/get-session-sigs.md

```diff
@@ -1,113 +1,331 @@
 
-## Resources You Can Request
+There are two parts to each ability request:
 
-You can pass an array of "resource ability requests" to the `getSessionSigs()` function, which will be presented to the user in the SIWE message - read more [here](resources-and-abilities) about Lit resources and abilities. The resources and abilities requested by the session key must be narrower or equal to the capabilities granted to it per the session capability object specified in the inner `AuthSig`. 
+#### `resource`
 
-When session capability objects are omitted from the `getSessionSigs()` function call, the SDK will generate a session capability object with **wildcard permissions against all of the resources in that category by default**, i.e. ability to perform operations against all access control conditions. Read more [here](capability-objects) about how to create custom session capability objects.
+This property is defining what Lit Resource we're granting the `ability` for. Each Lit Resource takes a resource identifier as an argument. You can pass `"*"` like so:
 
-## Clearing Local Storage
+```ts
+new LitPKPResource("*")
+```
 
-If you want to clear the session key stored in the browser local storage, you can call the [`disconnectWeb3` method](https://js-sdk.litprotocol.com/functions/auth_browser_src.ethConnect.disconnectWeb3.html).
+to grant an `ability` to all resources owned by the Ethereum address that authorized the request to generate Session Signatures for the Session Keys. In the case of this example, that would be the address corresponding to our `ETHEREUM_PRIVATE_KEY` constant (that we created an Ethers.js wallet for).
+
+Or you can specify a specific identifier for each resource as shown in the code snippets below.
+
+The available Lit Resource options are:
+
+##### `LitAccessControlConditionResource`
+
+Imported from the `@lit-protocol/auth-helpers` package:
+
+```ts
+import { LitAccessControlConditionResource } from "@lit-protocol/auth-helpers";
+```
+
+This resource specifies a Lit Access Control Conditions (ACC) the `ability` is being granted for. Specifying `"*"` grants the `ability` to any ACC, regardless of what the conditions are. To grant the `ability` for a specific ACC, you can specify the hashed key value of the ACC in place of `"*"` like so:
+
+<!-- TODO How to get the hashed key value of the ACC -->
+
+```ts
+new LitAccessControlConditionResource("")
+```
+
+##### `LitPKPResource`
+
+Imported from the `@lit-protocol/auth-helpers` package:
+
+```ts
+import { LitPKPResource } from "@lit-protocol/auth-helpers";
+```
+
+This resource specifies what Lit Programmable Key Pair (PKP) the `ability` is being granted for. Specifying `"*"` grants the `ability` for all PKPs. To grant the `ability` for a specific PKP, you can specify the token ID of the PKP in place of `"*"` like so:
+
```

Comment:
Should we warn that `*` permissions are basically no permissions? Since we will allow any pkp to use it if configured like this?